### PR TITLE
DT-2589 Move environment Phase up to header and a responsive feedback banner

### PR DESCRIPTION
**Pull request by andymarke**
Desktop View
Screenshot 2021-11-09 at 17 59 30

Smaller View with Feedback dropping to next line
Screenshot 2021-11-09 at 17 59 50

Mobile View with environment oases name shortened
Screenshot 2021-11-09 at 18 00 30

Pages without feedback survey more or less unchanged
Screenshot 2021-11-09 at 18 00 46

### CODE REVIEW

File: sass/components/_header.scss [hunk 1]
```diff
@@ -118,18 +132,22 @@
   @include govuk-width-container;
   @include govuk-responsive-margin(3, 'bottom');
   @include govuk-responsive-padding(3, 'top');
-  @include govuk-responsive-padding(3, 'bottom');
```
Comment (andymarke) [inline sass/components/_header.scss:121]: moved this down into the location bar and feedback, so when feedback wraps in next line we have a nice bottom padding

File: sass/components/_header.scss [hunk 2]
```diff
@@ -118,18 +132,22 @@
...
-  width: 100%;
```
Comment (andymarke) [inline sass/components/_header.scss:122]: this was incorrectly overriding the gov.uk width causing part of the bar to be hidden in smaller widths

File: sass/components/_header.scss [hunk 3]
```diff
@@ -140,6 +158,11 @@
     @include govuk-font($size: 19, $weight: 'normal');
   }
 
+  .app-phase-banner {
+    border-bottom: 0;
+    padding-bottom: 0;
+    padding-top: 0;
+  }
```
Comment (andymarke) [inline sass/components/_header.scss:165]: give no gap below and allow parent to deal with bottom padding

File: views/partials/header.njk [hunk 4]
```diff
@@ -13,6 +13,20 @@
...
+          classes: 'govuk-!-margin-right-1 ' + phaseNameColour
+        }) }}
+        </span>
+       <span class="dps-header__title__phase-short">
+        {{ govukTag({
+          text: phaseNameShort,
+          classes: 'govuk-!-margin-right-1 ' + phaseNameColour
+        }) }}
+        </span>
+      {% endif %}
```
Comment (andymarke) [inline views/partials/header.njk:29]: obviously only one  will be displayed of these depending on the screen width

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**